### ISSUE TITLE
Regex debugger tool single step ability.

### ISSUE THREAD

**Issue by swgmike** [firasdib/Regex101#1965]
## Feature
The regex debugger tool doesn't have the ability to single step thru the expression. It can only go to the beginning/end or play the regex expression. One must enter a max stop number to try a do a single step capability. Hence during play mode it may go beyond what is currently shown on the screen and by the time one scrolls down, it is already finished.

Also would be nice to select a particular expression within the regex to break point on.

**Comment by firasdib** [comment]
What do you mean? Have you tried using the arrow keys?

**Comment by swgmike** [comment]
I've used the arrow keys which are go to beginning or end, play, and go slower or faster. There is no button to step on increment at a time thru the regexpression at my speed so I can see each step at my speed to see if it is working correctly.
It would be nice to set a breakpoint based on the group # each time it is evaluated or on the group label name.

See attachment 
![firefox_yB8h8DBJ57](https://user-images.githubusercontent.com/60407871/210635257-78af08e8-b515-47b7-9644-6e81ed045d0e.png)

**Comment by firasdib** [comment]
That's weird. Have you tried clicking on `Available Keyboard Shortcuts` and trying everything in there?

**Comment by firasdib** [comment]
I will add a button in the UI for this.
